<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TransportBase</class>
 <widget class="QWidget" name="TransportBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>99</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MusE: Transport Panel</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>1</number>
   </property>
   <property name="spacing" >
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="recMode" >
       <item>
        <property name="text" >
         <string>Overdub</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Replace</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>6</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Rec Mode</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cycleMode" >
       <item>
        <property name="text" >
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Mix</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Replace</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>6</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Cycle Rec</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="punchinButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="icon" >
        <iconset resource="muse.qrc" >:/xpm/punchin.xpm</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loopButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="muse.qrc" >:/xpm/loop.xpm</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="punchoutButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="icon" >
        <iconset resource="muse.qrc" >:/xpm/punchout.xpm</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="arrowType" >
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="Awl::PosEdit" name="tl1" />
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>6</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Left Loc</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Awl::PosEdit" name="tl2" />
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>6</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Right Loc</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="Awl::PosEdit" name="time1" />
       </item>
       <item>
        <widget class="Awl::PosEdit" name="time2" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="slider" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>3</number>
       </property>
       <property name="spacing" >
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="tb1" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>rewind to start</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="muse.qrc" >:/xpm/start.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>rewind</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="muse.qrc" >:/xpm/frewind.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb3" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>forward</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="muse.qrc" >:/xpm/fforward.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb4" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>stop</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="muse.qrc" >:/xpm/stop.xpm</iconset>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb5" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>play</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="muse.qrc" >:/xpm/play.xpm</iconset>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb6" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>record</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="quantizeButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>AC</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clickButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Click</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="syncButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Sync</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>5</number>
     </property>
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>1</number>
        </property>
        <property name="spacing" >
         <number>1</number>
        </property>
        <item>
         <widget class="Awl::TempoEdit" name="tempo" />
        </item>
        <item>
         <widget class="SigLabel" name="sig" >
          <property name="palette" >
           <palette>
            <active>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>221</red>
              <green>223</green>
              <blue>228</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>85</red>
              <green>85</green>
              <blue>85</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
             <color>
              <red>86</red>
              <green>117</green>
              <blue>148</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>238</blue>
             </color>
             <color>
              <red>82</red>
              <green>24</green>
              <blue>139</blue>
             </color>
             <color>
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </active>
            <inactive>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>221</red>
              <green>223</green>
              <blue>228</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>85</red>
              <green>85</green>
              <blue>85</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
             <color>
              <red>86</red>
              <green>117</green>
              <blue>148</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>238</blue>
             </color>
             <color>
              <red>82</red>
              <green>24</green>
              <blue>139</blue>
             </color>
             <color>
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </inactive>
            <disabled>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>221</red>
              <green>223</green>
              <blue>228</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>85</red>
              <green>85</green>
              <blue>85</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>199</red>
              <green>199</green>
              <blue>199</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
             <color>
              <red>86</red>
              <green>117</green>
              <blue>148</blue>
             </color>
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
             <color>
              <red>0</red>
              <green>0</green>
              <blue>238</blue>
             </color>
             <color>
              <red>82</red>
              <green>24</green>
              <blue>139</blue>
             </color>
             <color>
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </disabled>
           </palette>
          </property>
          <property name="text" >
           <string>4/4</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="masterButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>use tempo map</string>
       </property>
       <property name="text" >
        <string>Tempomap</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Awl::PosEdit</class>
   <extends>QWidget</extends>
   <header>awl/posedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>SigLabel</class>
   <extends>QLabel</extends>
   <header>widgets/siglabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Awl::TempoEdit</class>
   <extends>QWidget</extends>
   <header>awl/tempoedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="muse.qrc" />
 </resources>
 <connections/>
</ui>
